benchmark/SoftMaxCrossEntropyBenchmark.jl

```diff
@@ -0,0 +1,63 @@
+include("../src/layers/LayerBase.jl")
+include("../src/layers/SoftMaxCrossEntropy.jl")
+include("../src/layers/SoftMax.jl")
+include("../src/layers/CrossEntropyLoss.jl")
+using Base.Test
+
+smaxcross = SoftMaxCrossEntropyLoss()
+smax = SoftMax()
+cross = CrossEntropyLoss()
+
+
+function before(smaxcross, smax, cross, dict)
+    init(smaxcross, nothing, dict)
+    init(smax     , nothing, dict)
+    init(cross    , nothing, dict)
+end
+
+function benchmmark(smaxcross, smax, cross, batch_size, input_size; alpha = 1.)
+  # batch_size = 10
+  # input_size = 5
+  before(smaxcross, smax, cross, Dict{String, Any}("batch_size" => batch_size, "input_size" => [input_size]))
+  x = rand(batch_size, input_size)
+  label = rand(0:input_size-1, batch_size,1)
+  # benchmark for original SoftMaxCrossEntropy
+  tic()
```

Comment:
I prefer a benchmark that could clearly tell me what's the execution time and allocations. Please check @times for it.